apps/showcase/src/app/app.component.ts

```diff
@@ -27,4 +27,8 @@ export class AppComponent {
   toggleSidenav(): void {
     this.sidenav.toggle();
   }
+
+  prettyJson(obj: unknown) {
```

Comment:
This should be a pipe in the long run. 🤔

Reply:
It turns out that regular `json` pipe is enough :)